src/Bicep.Core.Samples/Files/InvalidResources_CRLF/main.diagnostics.bicep

```diff
@@ -1925,6 +1928,7 @@ resource p6_res2 'Microsoft.Rp1/resource1/child2@2020-06-01' = {
 //@[017:060) [BCP081 (Warning)] Resource type "Microsoft.Rp1/resource1/child2@2020-06-01" does not have types available. (CodeDescription: none) |'Microsoft.Rp1/resource1/child2@2020-06-01'|
   parent: p6_res1
 //@[010:017) [BCP062 (Error)] The referenced declaration with name "p6_res1" is not valid. (CodeDescription: none) |p6_res1|
+//@[010:017) [BCP239 (Error)] Unexpected value for parent property. (CodeDescription: none) |p6_res1|

```

Comment:
Looking at the duplicate errors here, I think it would also be worth checking the type of the value, and ignoring it if it is `ErrorType`. This would avoid raising an extra error in the case where we've already raised an error.